packages/loader/container-definitions/src/runtime.ts

```diff
@@ -99,6 +99,8 @@ export interface IRuntime extends IDisposable {
      */
     setAttachState(attachState: AttachState.Attaching | AttachState.Attached): void;
 
+    getPendingLocalState(): unknown;
```

Comment:
I think at minimum we should comment all these APIs and mention that content has to be serializable.